gnowsys-ndf/gnowsys_ndf/ndf/management/commands/event_update_status.py

```diff
@@ -0,0 +1,71 @@
+from gnowsys_ndf.ndf.models import *
```

Comment:
We can do it hourly for previous events.Also for events which the user does not see in calendar, we should atleast change the status in the backend. We can change this frequency, from hourly to whatever is suitable.